modules/seating/copy.php

```diff
@@ -44,7 +44,7 @@
             $row['cols'],
             $row['name'] .' (Kopie)',
             $row['orientation'],
-            $row['u18'],
+            empty($row['u18']) ? $row['u18'] : '0',
```

Comment:
Is this check correct?
It says If $row['u18'] is empty, use $row['u18'], otherwise 0.

Why should we use $row['u18'], if $row['u18'] is empty?

Reply:
Good catch. This is the wrong way around.
Fixed it.